src/main/webapp/loancalc.jsp

```diff
@@ -0,0 +1,90 @@
+<%@ page contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+<%@taglib prefix="t" tagdir="/WEB-INF/tags" %>
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@page errorPage="error.jsp" isErrorPage="false" %>
+
+<%-- DEV: This JSP page is used to display the visuals of the Loan Calculator.
+ It displays a form which takes positive numbers as input from the user and calculates the cost, interest with more
+ and displays the new information in a table in the bottom of the page. --%>
+
+<t:pagetemplate>
+    <jsp:attribute name="header">
+             Loan Calculator
+    </jsp:attribute>
+
+     <jsp:body>
+       <%-- <t:info>
+           <h1 class="card-title> Loan calculator </h1>
+               <p class="card-text"> This program is designed to show you variations of a loan you have taken or maybe
+           will take in the future. The loan calculator is designed to calculate different aspects of your loan
+           including, but not limited to, the total cost of the loan, and your interest rate. The program will also
+           showcase different prices with a shorter/longer loan period or even a lower APRC. </p>
+        <%-- </t:info> --%>
+
+
+        <%-- <t:content> --%>
+         <%-- Form for user input --%>
+        <form action="loancalc" method="POST">
+            <div class="form-group">
+                <label for="loanAmount">Loan amount</label>
+                <input type="number" class="form-control" id="loanAmount" name="loanAmount" min="1" step="0.01" placeholder="Enter loan amount">
+            </div>
+            <div class="form-group">
+                <label for="APRC">APRC</label>
+                <input type="number" class="form-control" id="APRC" name="APRC" min="0.1" step="0.01" aria-describedby="APRChelp" placeholder="Enter APRC">
+                <small id="APRChelp" class="form-text text-muted">Annual Percentage Rate of Charge (ÅOP)</small>
+            </div>
+            <div class="form-group">
+                <label for="duration">Duration in months</label>
```

Comment:
Note. hvis tallet er for højt, giver den NaN eller not a number, tilbage.
Ikke noget der burde fiksets, men havde det som en mente, af mulig fejl